styles/_colors.scss

```
@@ -13,6 +13,9 @@ $gray-700: #343437;
 $gray-800: #232325;
 $gray-900: #111112;
 $black:    #000000;
+$black-10: #F5F5F7;
+$black-40: #A2A2A4;
+$black-60: #454549;
```

These are just aliases for `$gray-100, $gray-400, and $gray-600`, right? Can we do without them? If not, I suggest explicitly defining them in terms of the original name, like this:

```suggestion
$black-10: $gray-100;
$black-40: $gray-400;
$black-60: $gray-600;
```